src/SourceBuild/content/eng/tools/tasks/PackageDiff/PackageDiff/PackageDiff.csproj

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```

Comment:
I don't think that we need a CLI frontend. The MSBuild task should be enough. We don't have that for the other analysis types either (poison check, prebuilt check, etc.)

Reply:
Yeah, I had thought we needed the task to be netstandard2.0, which didn't have the reflection API's to get the assembly info. It looks like this task can be just netcurrent, so I can put everything in the task assembly and simplify this a bit.

Reply:
Yes, this doesn't need to support desktop msbuild.